ignite/pkg/cosmosclient/cosmosclient_test.go

```diff
@@ -79,34 +79,32 @@ func newClient(t *testing.T, setup func(suite), opts ...cosmosclient.Option) cos
 }
 
 func TestNew(t *testing.T) {
-	assert := assert.New(t)
-
```

Comment:
Not convinced by this change :thinking:... why ?

Reply:
Just for consistency with the rest of the codebase.  The functionality is the same either way.  We can also just remove usage of `assert` altogether and use `require.SOMETHING(`t, ...)` for all testing.

Reply:
`assert` and `require` have different purposes, I tend to use both. I use `require` when the assertion is strong enough to stop the test, like an unexpected error, and `assert` for the others, because it doesn't interrupt the test. Like that a test result can have multiple assertions fails.

Using `assert.New` and `require.New` makes assertions easier to read and write, so sorry even for code consistency I'm not going to stop using them :P 

I resolve the conversation anyway.